packages/mongodb-rag-core/src/contentStore/MongoDbEmbeddedContentStore.ts

```diff
@@ -274,6 +353,36 @@ export function makeMongoDbEmbeddedContentStore({
       }
     },
 
+    async listDataSources(): Promise<DataSourceMetadata[]> {
+      const now = Date.now();
+      const maxAge = 1000 * 60 * 60 * 24 * 7; // 7 days
```

Comment:
for this and the other magic numbers, pls define as constants at the top of the file